### DISCUSSION TITLE
Olá sou Carlos e querias umas dicas de recursos e repositórios algus para eu aprender sobre Elixir

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2002] [Perguntas e Respostas] [answered]
# Olá sou Carlos e querias umas dicas de features e algus repositórios para eu aprender sobre Elixir

**Comment by victorferreira** [comment]
Se o inglês não for uma barreira, o melhor lugar da internet pra fazer perguntas sobre Elixir é o [fórum da própria linguagem](https://elixirforum.com). Quem sabe o próprio Valim não responde sua pergunta por lá...

**Reply by ghost** [reply]
Obrigado ai

**Comment by wmartins** [comment, marked as answer]
Eu acho o site do próprio Elixir muito bem feito. O [_Getting Started_](https://elixir-lang.org/getting-started/introduction.html) vai te guiando com vários exemplos e tu pode ir acompanhando junto.

O [tutorial sobre OTP](https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html) eu achei fantástico quando fiz. Consegui aprender bastante coisa por lá.

E acho que a dica do @victorferreira do forum é muito boa. Tem várias boas discussões por lá e pelo que vejo é uma comunidade bem ativa e com um monte de gente legal.

Sobre repositórios, aí acho que vai mais do teu interesse. No geral, conforme tu for ficando mais familiarizado com a linguagem/plataforma, acaba sendo bem fácil de ler os repositórios de alguma library, por exemplo.

**Reply by ghost** [reply]
Obrigado cara, tava precisando mesmo.

**Comment by vitebo** [comment]
@CarlosVitr nessa próxima [next level week](https://nextlevelweek.com/inscricao/4) vai ter uma trilha de elixir, talvez ajude

**Comment by TheyCallMeErick** [comment]
Acho que a elixir school pode ser uma boa para aprender, está bem documentada e ainda por cima, está traduzida
[https://elixirschool.com/pt/](url)
